sdks/python/apache_beam/runners/interactive/utils_test.py

```diff
@@ -42,6 +42,30 @@
 from apache_beam.utils.timestamp import Timestamp
 from apache_beam.utils.windowed_value import WindowedValue
 
+# Protect against environments where apitools library is not available.
+try:
+  from apitools.base.py.exceptions import HttpError
+  from apitools.base.py.exceptions import HttpNotFoundError
+except ImportError:
```

Comment:
```
except ImportError:
    _http_error_imported = False
    HttpError = ValueError
    HttpNotFoundError = ValueError
else:
    _http_error_imported = True


@unittest.skipIf(
    not _http_error_imported,
    'http errors are not imported.')
class GCSUtilsTest...
```

Reply:
I've added this code now, these unit tests should be skipped now if _http_error_imported is False.